includes/Core/REST_API/REST_Routes.php

```diff
@@ -167,69 +167,6 @@ private function get_routes() {
 		};
 
 		$routes = array(
-			// TODO: This route is super-complex to use and needs to be simplified.
```

Comment:
We also need to do this:

> * Edit `REST_Routes` class:
>   * **Delete the `error_to_response` method.**

Reply:
https://github.com/google/site-kit-wp/pull/3755/commits/47134117766ffd6a887f60ee03f4eccef24f4f33